plugins/reports/counter/classes/releases/CounterRelease.inc.php

```diff
@@ -0,0 +1,254 @@
+<?php
+
+/**
+ * @file plugins/generic/sushiLite/classes/releases/CounterRelease.inc.php
+ *
+ * Copyright (c) 2014 University of Pittsburgh
+ * Distributed under the GNU GPL v2 or later. For full terms see the file docs/COPYING.
+ *
+ * @class CounterRelease
+ * @ingroup plugins_reports_counter
+ *
+ * @brief A COUNTER report, base class
+ */
+require_once(dirname(dirname(__FILE__)).'/COUNTER/COUNTER.php');
+
+define('COUNTER_EXCEPTION_WARNING', 0);
+define('COUNTER_EXCEPTION_ERROR', 1);
+define('COUNTER_EXCEPTION_PARTIAL_DATA', 4);
+define('COUNTER_EXCEPTION_NO_DATA', 8);
+define('COUNTER_EXCEPTION_BAD_COLUMNS', 16);
+define('COUNTER_EXCEPTION_BAD_FILTERS', 32);
+define('COUNTER_EXCEPTION_BAD_ORDERBY', 64);
+define('COUNTER_EXCEPTION_BAD_RANGE', 128);
+define('COUNTER_EXCEPTION_INTERNAL', 256);
+
+class CounterRelease {
```

Comment:
I would call it just CounterReport. Counter Release just adds extra noise that's not much important for a general class. Your other more specific classes (CounterRelease3, for example) can bring the extra piece of information right next to where it's implemented.